src/services/auth.service.js

```diff
@@ -0,0 +1,63 @@
+// AUTH-IMPORT: Import Arcana auth
+// ...
+
+function createAuthService() {
+  // AUTH-1: Create a new instance of Arcana AuthProvider.
+  // ...
+
+  async function init() {
+    // AUTH-2: Initialize the wallet with App Mode (UI / NoUI).
+    // ...
```

Comment:
add something like this - `const wallet ....`

Reply:
Resolved at https://github.com/arcana-network/sdk-demo/pull/5/commits/c6a0fe0dfbe2f54999bd47a2a35c6fa3d7b63447